test/telemetry_metrics_appsignal_test.exs

```diff
@@ -145,6 +145,54 @@ defmodule TelemetryMetricsAppsignalTest do
     assert_receive {^ref, :called}
   end
 
+  test "reporting calculated metrics with unary functions" do
+    # `Telemetry.Metrics.Summary` maps to AppSignal's measurement metric
+    metric =
+      summary("db.query.foo",
```

Comment:
I think we could just use a proper name:

```suggestion
      summary("db.query.duration",
```

Reply:
Done!

Reply:
I think you missed it here.

```suggestion
      summary("db.query.duration_multiplied",
```